src/lib/resources.ts

```diff
@@ -9,7 +9,7 @@ export const mintIngestorResources = (): MintIngestorResources => {
     }
     const settings = {
       apiKey: ALCHEMY_API_KEY,
-      network: Network.BASE_MAINNET, // Replace with the correct network
+      network: Network.ETH_MAINNET, // Replace with the correct network
```

Comment:
Ah, this is not safe

I prepared this update to handle this scenario:
https://github.com/floornfts/mobile-minting/pull/48

I'm rebasing it against main now to see if it will merge cleanly & then you should be able to use it

Reply:
(This change was global to all ingestors & broke all the other Base ingestors 🤣 )

Reply:
I will update it once it is merged.

Reply:
I will take a closer look and update the codebase, since opening this PR they implemented a couple of new features.
It looks like the contract's [renderer](https://github.com/visualizevalue/mint/blob/main/contracts/contracts/renderers/P5Renderer.sol) has a JS script that decodes the nft's assets.